test/SILOptimizer/di_property_wrappers.swift

```diff
@@ -535,6 +535,54 @@ func testSR_12341() {
   _ = SR_12341(condition: true)
 }
 
+class TestLeak: CustomStringConvertible {
```

Comment:
Optional: you might want to stick a LifetimeTracked in here as well? StdlibUnittest will automatically assert if one leaks.